src/components/Person/Person.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${name}`}</h2>
+    {age && (<p className="Person__age">{`I am ${age}`}</p>)}
+    <p className="Person__partner">
+      {isMarried ? (
+        `My ${sex === 'f' ? `husband's` : `wife's`} name is ${partnerName}`
+      ) : (
+        `I am not married`
+      )}
```

Comment:
One follow-up comment.
Complex logical operations better to be done not in function return. If necessary - create variables and use them in html.